www/js/app.js

```diff
@@ -17,6 +13,7 @@ var app = angular.module('starter',
             }
         });
     })
+
```

Comment:
Again, watch out for these whitespace changes, especially for dot-chained functions there should not be a new line in the chain